valkey.conf

```diff
@@ -386,6 +386,11 @@ databases 16
 # ASCII art logo in startup logs by setting the following option to yes.
 always-show-logo no
 
+# To avoid logging personal identifiable information (PII) into server log file,
+# uncomment the following:
```

Comment:
I think PII is the wrong way to frame this. We should probably label it as something like `user data`, which could contain PII.